assets/css/_sass/website/_home.scss

```diff
@@ -132,6 +132,14 @@
         text-align: center;
       };
     }
+    /**
+      * Overrides for homepage buttons
+    */
+    .hxBtn {
+      a {
```

Comment:
check toolkit

Reply:
See comment on index.md that deals with this issue.